Sources/FaceLivenessDetection/UI/Components/FaceDetectionViewController.swift

```diff
@@ -342,29 +342,30 @@ extension FaceDetectionViewController: AVCaptureDataOutputSynchronizerDelegate {
         let faceBoundingBox = face.boundingBox
         let faceArea = faceBoundingBox.width * faceBoundingBox.height
 
-        var instruction: FaceDetectionState
         if faceArea < 0.15 {
-            instruction = .faceTooFar
+            DispatchQueue.main.async {
+                self.faceDetectionViewModel.instruction = .faceTooFar
+            }
         } else if faceArea > 0.25 {
-            instruction = .faceTooClose
+            DispatchQueue.main.async {
+                self.faceDetectionViewModel.instruction = .faceTooClose
+            }
         } else if abs(yaw) > 0.08 {
-            instruction = .faceFront
-//            if yaw > 0 {
-//                instruction = .faceRight
-//            } else {
-//                instruction = .faceLeft
-//            }
+            DispatchQueue.main.async {
+                self.faceDetectionViewModel.instruction = .faceFront
+            }
         } else {
-            instruction = .faceFit
+            analyzeFaceLiveness()
         }
 
-        guard instruction != faceDetectionViewModel.instruction else { return }
-        DispatchQueue.main.async {
-            self.faceDetectionViewModel.instruction = instruction
-        }
+        // prevent view from frequently rerendering
+//        guard instruction != faceDetectionViewModel.instruction else { return }
+//        DispatchQueue.main.async {
+//            self.faceDetectionViewModel.instruction = instruction
+//        }
```

Comment:
Remove commented code if we don't need it

Reply:
fixed